Makefile

```diff
@@ -185,34 +193,34 @@ lint-api-schema: build-api-schema
 	$(IN_VENV) codespell -I .ci/ignore-spelling.txt _schema.yaml
 	$(MAKE) remove-api-schema
 
-client: node-deps ## Rebuild client-side artifacts for local development.
+install-client: node-deps ## Install prebuilt client as defined in root package.json
+	yarn install && yarn run stage
+
+client: client-node-deps ## Rebuild client-side artifacts for local development.
 	cd client && yarn run build
 
-client-production: node-deps ## Rebuild client-side artifacts for a production deployment without sourcemaps.
+client-production: client-node-deps ## Rebuild client-side artifacts for a production deployment without sourcemaps.
 	cd client && yarn run build-production
 
-client-production-maps: node-deps ## Rebuild client-side artifacts for a production deployment with sourcemaps.
+client-production-maps: client-node-deps ## Rebuild client-side artifacts for a production deployment with sourcemaps.
 	cd client && yarn run build-production-maps
 
-client-format: node-deps ## Reformat client code
+client-format: client-node-deps ## Reformat client code
 	cd client && yarn run format
 
-client-watch: node-deps ## A useful target for parallel development building.  See also client-dev-server.
```

Comment:
There's one case I like to use watch for, which is running/debugging selenium tests and changing the client as I go. Sorry that I didn't think about this when you asked. I should be able to target the test driver to a predefined port though, so that should also work with the dev server.